### ISSUE TITLE
What is your version of arcface_checkpoint.tar?

### ISSUE THREAD

**Issue by ygtxr1997** [neuralchen/SimSwap#343]
I replaced your 'arcface-checkpoint.tar' with arcface-100 pretrained on MS1M.
After training SimSwap, I found that the identity similarity seems largely worse than your provided SimSwap-224.

Target, source, and swapped result are shown below:
![target](https://user-images.githubusercontent.com/22703453/202655994-d4753d90-4057-453c-ab11-cd6a621f2b96.jpg)
![source](https://user-images.githubusercontent.com/22703453/202656004-381c97c5-1bc1-4797-b0be-4d1caddba92e.jpg)
![result](https://user-images.githubusercontent.com/22703453/202656013-82467208-54d3-4ed2-85c5-4a83cfb40159.jpg)

**Comment by ygtxr1997** [comment]
> I think they took the checkpoint from this repository. It is possible that if the model was initially trained using arcface-100 pretrained on MS1M, then the result would be better (but this is not certain)
> 
> https://github.com/foamliu/InsightFace-v2/releases/tag/v1.0

Thanks for your answer!
I'm not sure if I mistakenly set the error backward propogation during training.
I used the same model codes (generator and discriminator) but re-wrote the training codes using pytorch-lightning by myself, including dataloader, loss function, and so on.
Maybe I need to check the training logic and receipe more carefully.
I show my training losses below, can you find some possible mistakes in these metrics?
![image](https://user-images.githubusercontent.com/22703453/202994450-46cbb58d-26e2-495d-bd8e-c6c4c495c451.png)
![image](https://user-images.githubusercontent.com/22703453/202994490-3aec0692-423f-4e20-818d-b9178d2f3f1b.png)
![image](https://user-images.githubusercontent.com/22703453/202994537-689b46c3-d221-4776-919d-dc45e54ecc83.png)

**Comment by ygtxr1997** [comment]
In my case, the mistaken face alignment scheme is probally responsible for the identity inconsistency issue. Closing.

**Comment by renmengyuan** [comment]
> In my case, the mistaken face alignment scheme is probally responsible for the identity inconsistency issue. Closing.

Hi， what kind of face alignment scheme is better when you train the model?

**Comment by GeneralShan** [comment]
simswap new data is ffhq
